packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStep.tsx

```diff
@@ -157,13 +86,13 @@ export const FormStep = ({
                     {fields.length === 0 && (
                         <Center
                             onDrop={item => {
-                                // We don't want to drop steps inside of steps
-                                if (item.ui === "step") {
-                                    return undefined;
-                                }
-                                handleDropField(item, {
-                                    row: 0,
-                                    index: 0
+                                composeHandleDropParams({
```

Comment:
Let's refactor this. 

Can we move these `onDrop` callbacks above L66? Into useCallback?

And, while doing that, give them meaningful names?

Basically what we did for example [here](https://github.com/webiny/webiny-js/pull/3574/files#diff-95bbc970b428ba6ab7d02726c08eac78aa2b0121ad5f3f65a4ac6e1a0fcce320R101).

it's because this code here is hard to read.